core/src/main/java/io/ddf/DDFManager.java

```diff
@@ -89,6 +88,10 @@ public void removeDDF(DDF ddf) throws DDFException {
     mDDFs.remove(ddf.getUUID());
   }
 
+  public DDF[] listDDFs() {
```

Comment:
@Huandao0812 will this return DDF without name as well? 
Also have you talked with Nhan/Vu to update his API.
Best way is to create task on jira for them and related it to your ticket.